docs/docs/Components/components-logic.md

```diff
@@ -241,6 +216,80 @@ Your flow should now look like this:
 |--------------|-------------|---------------------------------------------------------------|
 | run_outputs  | A `List` of types `Data`, `Message,` or `DataFrame`  | All outputs are generated from running the flow.                   |
 
+## Legacy components
+
+## Data Conditional Router
+
+:::important
+This component is in **Legacy**, which means it is no longer in active development as of Langflow version 1.3.
+:::
+
+This component routes `Data` objects based on a condition applied to a specified key, including boolean validation. It can process either a single Data object or a list of Data objects.
+
+This component is particularly useful in workflows that require conditional routing of complex data structures, enabling dynamic decision-making based on data content.
+
+### Inputs
+
+| Name          | Type     | Description                                                                       |
+|---------------|----------|-----------------------------------------------------------------------------------|
+| data_input    | Data     | The Data object or list of Data objects to process. Can handle both single items and lists. |
+| key_name      | String   | The name of the key in the Data object to check.                                  |
+| operator      | Dropdown | The operator to apply. Options: "equals", "not equals", "contains", "starts with", "ends with", "boolean validator". Default: "equals". |
+| compare_value | String   | The value to compare against. Not shown/used when operator is "boolean validator". |
+
+### Outputs
+
+| Name         | Type        | Description                                          |
+|--------------|-------------|------------------------------------------------------|
+| true_output  | Data/List   | Output when the condition is met.                    |
+| false_output | Data/List   | Output when the condition is not met.                |
+
+### Operator Behavior
+
+- **equals**: Exact match comparison between the key's value and compare_value
+- **not equals**: Inverse of exact match
+- **contains**: Checks if compare_value is found within the key's value
+- **starts with**: Checks if the key's value begins with compare_value
+- **ends with**: Checks if the key's value ends with compare_value
+- **boolean validator**: Treats the key's value as a boolean. The following values are considered true:
+  - Boolean `true`
+  - Strings: "true", "1", "yes", "y", "on" (case-insensitive)
+  - Any other value is converted using Python's `bool()` function
+
+### List Processing
+
+The following actions occur when processing a list of Data objects:
+- Each object in the list is evaluated individually
+- Objects meeting the condition go to true_output
+- Objects not meeting the condition go to false_output
+- If all objects go to one output, the other output is empty
+
+## Deprecated components
```

Comment:
You should not have a heading without any body text. Add something to describe what this section is about.